src/hyperlight_testing/src/lib.rs

```diff
@@ -129,3 +130,28 @@ pub fn c_callback_guest_as_string() -> Result<String> {
         .map(|s| s.to_string())
         .ok_or_else(|| anyhow!("couldn't convert callback guest PathBuf to string"))
 }
+
+/// Get a fully qualified path to a simple guest binary preferring a binary
+/// in the same directory as the parent executable. This will be used in
+/// fuzzing scenarios where pre-built binaries will be build and submitted to
+/// a fuzzing framework.
+pub fn simple_guest_for_fuzzing_as_string() -> Result<String> {
+    let exe_dir = env::current_exe()
+        .ok()
+        .and_then(|path| path.parent().map(|p| p.to_path_buf()));
+
+    if exe_dir.is_some() {
```

Comment:
is this `.is_some()` redundant given the next line?

Reply:
yes, i think is is :P

Reply:
fixed